### DISCUSSION TITLE
PurchaseState alway 0?

### DISCUSSION THREAD

**Discussion by ChrAus** [imdhemy/laravel-in-app-purchases#96] [Q&A] [answered]
Hi guys,

I got this package to work and started checking the recipes in my app.
They all seem to have purchaseState = 0.

How do I check if those products were paid properly?
Any help is appreciated.

```
Imdhemy\GooglePlay\Products\ProductPurchase {#2264
  #kind: "androidpublisher#productPurchase"
  #purchaseTimeMillis: "1634409867148"
  #purchaseState: 0
  #consumptionState: 1
  #developerPayload: ""
  #orderId: "GPA.xxxx-xxxx-xxxx-xxxx"
  #purchaseType: null
  #acknowledgementState: 1
  #purchaseToken: null
  #productId: null
  #quantity: null
  #obfuscatedExternalAccountId: null
  #obfuscatedExternalProfileId: null
  #regionCode: "DE"
}
```

**Comment by imdhemy** [comment, marked as answer]
@ChrAus 
For **Google Play**, a successful response means a valid receipt. As simple as that.

**Reply by ChrAus** [reply]
Thanks, that's straightforward

**Reply by imdhemy** [reply]
@ChrAus My pleasure!

BTW this discussion may be helpful for you #24
